src/spotify-pupeteer/index.js

```diff
@@ -131,19 +131,19 @@ async function postEpisode(youtubeVideoInfo) {
     });
   }
 
-  async function anchorLogin() {
+  async function spotifyLogin() {
     logger.info('-- Accessing Spotify for Podcasters login page');
     await clickSelector(page, '::-p-xpath(//button[contains(text(), "Continue")])');
 
     logger.info('-- Logging in');
     /* The reason for the wait is because
-    anchorfm can take a little longer to load the form for logging in
+    spotifyfm can take a little longer to load the form for logging in
     and because pupeteer treats the page as loaded(or navigated to)
     even when the form is not showed
     */
     await page.waitForSelector('#email');
-    await page.type('#email', env.ANCHOR_EMAIL);
-    await page.type('#password', env.ANCHOR_PASSWORD);
+    await page.type('#email', env.SPOTIFY_EMAIL);
```

Comment:
We're introducing a breaking change and so we should be explicit in the doc that the latest version people need to adapt

Reply:
Where is the correct section to specify this change?

Reply:
In the readme, where we are implementing the documentation of these .env variables

Reply:
Done